icaruscode/Timing/timing_corrections_tags.fcl

```diff
@@ -24,4 +24,14 @@ PMTtimingCorrectionsTags_Run2_August2023: {
   CosmicsTag:   "v1r0"  # tables for run>=0 (null) and run>=8046 
 }
 
+# These are the standard tags for analyses on Run 1 and Run 2 data (as of September 2023)
+# These tagged versions of the databases only contain tables relevant for Run 1 and Run 2.
+# Notes: 
+#  - No cosmics corrections are available for Run 2
+PMTtimingCorrectionsTags_Run2_Sept2023: {
+  CablesTag:    "v2r1"  # tables for run>=0 (null), run>=8046, run>=9301, run>=9628 and run>=9773
+  LaserTag:     "v2r1"  # tables for run>=0 (null), run>=8046, run>=9301, run>=9628 and run>=9773
+  CosmicsTag:   "v1r0"  # tables for run>=0 (null) and run>=8046 
```

Comment:
This implies that we are applying on Run2 inconsistent laser and cosmic ray corrections.
Ideally we would have a null cosmic-ray-based correction for `run>=9301`.

Reply:
Yes, it would be better. It would also simplify my work in extracting the cosmics corrections for Run 2 by not having to subtract already applied cosmics corrections (or having to ask to specifically produce calibration ntuples with cosmics corrections disabled). I'll ask @jedori0228 how painful it would be to add it...